docs/installing-solidity.rst

```diff
@@ -60,17 +60,39 @@ Please refer to the solc-js repository for instructions.
 Docker
 ======
 
-We provide up to date docker builds for the compiler. The ``stable``
-repository contains released versions while the ``nightly``
-repository contains potentially unstable changes in the develop branch.
+Docker images of Solidity builds are available using the ``solc`` image from the ``ethereum`` organisation. Use the ``stable`` tag for the
```

Comment:
"using the ``solc`` image" sounds a little like the builds are created with the help of the ``solc`` image.